public/app/percona/shared/components/Elements/TabbedContent/TabbedContent.types.ts

```diff
@@ -0,0 +1,22 @@
+export interface ContentTab {
+  label: string;
+  key: string;
+  disabled?: boolean;
+  component: () => JSX.Element;
```

Comment:
Why do we need a function here? Why not just `JSX.Element` or `ReactNode`

Reply:
You're right, thanks 👍